.changelog/v0.32.0/summary.md

```diff
@@ -0,0 +1,14 @@
+This release notably comes with a fully featured [light client attack detector][attack-detector],
```

Comment:
```suggestion
*May 3, 2023*

This release notably comes with a fully featured [light client attack detector][attack-detector],
```

We actually also need to go back through previous releases' summaries and add release dates there too.

Reply:
https://github.com/informalsystems/tendermint-rs/pull/1314/commits/64a4904460872a3f05b7052dcef4d15d21bd4da6